book.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'rental'
+
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    rental.book = self
+    @rentals << rental
+  end
```

Comment:
In order to achieve the `has-many` associations, the person itself should be present as a parameter to create the new rental. Kindly consider creating a new rental in this method for adding rentals and pass the date and person as parameters. It should be similar to something below: 

       def add_rental(person, date)
          Rental.new(date, self, person)
       end